ChiantiPy/Gui/gui_cl/gui.py

```diff
@@ -13,7 +13,7 @@ def chpicker(path,  filter='*.*', label=None):
     the label keyword is included for consistency but does nothing'''
     if not os.path.isdir(path):
         path=os.curdir
-#        if type(pattern) == NoneType:
+#        if type(pattern) is NoneType:
```

Comment:
Would `if pattern is None:` not be sufficient? (same at other places)

Reply:
Well, I think it should be. I'll update these!

Reply:
I think ebuchlin is right.  If you could update then that would be great

thanks for going over the code.
Ken

Reply:
On second thoughts, I think that since this is a comment describing the behavior, the comment should stay as it is. I find it more descriptive that way. :sweat_smile: 
Still, if you think it should be changed, let me know!

Reply:
Thank you for letting me participate! :smile: